core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
Yea, we'd also need to make people aware in the release notes of the change.

I think you could easily just do `include Spree::Core::NumberGenerator.new(prefix: 'C', length: 9)` in your decorator though, but not sure whether including it twice is the best idea.